src/cmd/contact/trust.sh

```diff
@@ -35,8 +35,9 @@ cmd_contact_trust() {
         *) fail "Unknown trust level: $level" ;;
     esac
 
-    local commands=$(echo "trust|$level|quit" | tr '|' "\n")
-    call_gpg fn/interact.py $contact $commands
+    # blank quotes "" => None
+    local commands=("" trust "" $level "" quit "" "")
+    call_gpg fn/interact.py "$contact" "${commands[@]}"
```

Comment:
Here you are complicating the code by using Bash arrays, they are not the best or strongest feature of Bash. I think that it is better to pass the commands as a string (as it was done before), and you can use `commands.split("\n")` in Python to convert it to an array. I think this is cleaner.

Actually, I think that you should give only the action (for example set-trust) and the `$level` of trust as arguments. The python code should figure out the sequence of commands that needs to be sent interactively to perform the given action.